cms/topics/tests/test_viewsets.py

```diff
@@ -224,3 +226,164 @@ def test_highlighted_methodology_viewset_configuration(self):
         )
         self.assertEqual(highlighted_methodology_page_chooser_viewset.edit_item_text, "Edit Methodology page")
         self.assertIn("topic_page_id", highlighted_methodology_page_chooser_viewset.preserve_url_parameters)
+
+
+class SeriesWithHeadlineFiguresChooserViewSetTest(WagtailTestUtils, TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.superuser = cls.create_superuser(username="admin")
+
+        # Create test pages
+        cls.topic_page = TopicPageFactory(title="PSF")
+        cls.series1 = ArticleSeriesPageFactory(parent=cls.topic_page, title="Series A")
+        cls.series2 = ArticleSeriesPageFactory(parent=cls.topic_page, title="Series B")
+
+        cls.series1_article = StatisticalArticlePageFactory(
+            parent=cls.series1,
+            title="Article 1",
+            release_date=datetime(2024, 1, 1),
+        )
+        cls.series2_article = StatisticalArticlePageFactory(
+            parent=cls.series2,
+            title="Article 2",
+            release_date=datetime(2024, 2, 1),
+        )
+
+        cls.series1_article.headline_figures = [
```

Comment:
nitpick: you can pass these to the factory as a kwarg

Reply:
Done in [04775c3](https://github.com/ONSdigital/dis-wagtail/pull/181/commits/04775c36e01e51ddc52454643fd30027089c2550)